src/sys/socket/sockopt.rs

```diff
@@ -574,14 +574,34 @@ sockopt_impl!(
     super::XuCred
 );
 #[cfg(apple_targets)]
+sockopt_impl!(
+    /// Get the credentials of the peer process of a connected unix domain
+    /// socket.
+    LocalPeerCred,
+    GetOnly,
+    libc::SOL_LOCAL,
```

Comment:
I was thinking change 0 of the above FreeBSD impl to `libc::SOL_LOCAL` as well, but then I realized that the libc crate does not have it defined. 😢

Reply:
Yes, that's what I did first in order to see how the build in CI would respond, but fell back to this inlining of sorts. This part is really optional though, I just thought it could help clean things up a bit and make them more readable, so it can be grafted away if need be.